### DISCUSSION TITLE
Managed MySQL/MariaDB/Percona

### DISCUSSION THREAD

**Discussion by thecodeassassin** [ubicloud/ubicloud#1414] [Ideas]
Hi there,

I love this project but we are sorely missing managed MySQL (or variants). The reason for this is because our projects demand both MySQL and PostgreSQL unfortunately. We have some legacy components that depend on MySQL as does our wordpress instance.

Right now we kind of depend on ClusterControl which isn't amazing because the free version does not allow us to scale or make backups so we end up still relying on Snapshooter for this. Also it would be nice to manage everything from a single pane of glass.

**Comment by byucesoy** [comment]
Hi @thecodeassassin,

At the moment, we don't have any timeline for MySQL or its variants. With the goal of being an open source cloud provider, I think at some point, we would have other databases than PostgreSQL, however there are other competing priorities such as K8s or app service. Depending on the number of asks, we can reevaluate the priority of MySQL and other databases.

**Comment by DrZoidberg09** [comment]
@byucesoy One year later I would like to ask the question again. We have an infrastructure at Hetzner with both MariaDB and Postgres. We would like to transfer it to a managed version. The lack of MySQL/MariaDB is the only reason not to do it yet. Hence the question: Any update on the timeline for MariaDB?

**Reply by byucesoy** [reply]
Hi @DrZoidberg09,

Unfortunately, there is no timeline for managed MySQL/MariaDB. It is not in our immediate focus, so I believe it wouldn't be available in short/medium term.  You can start using our managed Postgres though. Is there any blocker for that?

**Reply by DrZoidberg09** [reply]
One key element of our services relies on MariaDB with not Postgres support coming soon. Still considering to migrate to Ubicloud, if we find a good solution for MariaDB, because the rest of your offering Postgres, Kubernetnets etc. is very appealing.
Thanks for the quick answer!
